### ISSUE TITLE
testTestSuiteExists  failing in Bazel release 0.11.0rc3

### ISSUE THREAD

**Issue by katre** [bazelbuild/bazel-integration-testing#17]
https://ci.bazel.build/blue/organizations/jenkins/Global%2Fbazel-integration-testing/detail/bazel-integration-testing/212/tests

Platforms:
- darwin-x86_64

**Comment by katre** [comment]
Blocking release https://github.com/bazelbuild/bazel/issues/3959

**Comment by ittaiz** [comment]
This looks really bazel core. But doesn't seem related to the two cherry picks.
It's trying to call `java` to build a jar.
Can you re run it? 
On a related note- do you know who's responsible for ci.bazel.build? I'd like to get a user there for example so I can re-run tasks

**Comment by katre** [comment]
The core error seems to be:
```
java.io.IOException: Cannot run program \"/private/var/tmp/_bazel_ci/4b9a32a7299075b6d80804c042d79b47/bazel-sandbox/2661807843882205/execroot/build_bazel_integration_testing/_tmp/8259161321b688c625eac361a1fc0e2e/26d3d1bbdb875e9b656d5a085df9de01/execroot/build_bazel_integration_testing/external/local_jdk/bin/java\" (in directory \"/private/var/tmp/_bazel_ci/4b9a32a7299075b6d80804c042d79b47/bazel-sandbox/2661807843882205/execroot/build_bazel_integration_testing/_tmp/8259161321b688c625eac361a1fc0e2e/26d3d1bbdb875e9b656d5a085df9de01/execroot/build_bazel_integration_testing\"): error=2, No such file or directory"
```

So possibly there's a missing dependency only on Darwin?

**Comment by katre** [comment]
Also, @philwo is the person to ask about ci.bazel.build.

**Comment by ittaiz** [comment]
@katre do you think the failure is related to rc3 or it surfaced something by chance?
Because I'm not sure how the two cherry-picks from rc2 (which triggered a green run) could have that affect. I might be misunderstanding the `remote` change.
If it's by chance- do you know if and where I can find test history and fail percentage on jenkins?

**Comment by katre** [comment]
Jenkins is pretty bad with test history, the page exists (https://ci.bazel.build/job/bazel-integration-testing/test_results_analyzer/) but frequently times out for me.

I suspect this is a flake, it looks suspiciously similar to https://github.com/bazelbuild/bazel/issues/4627, which is also not finding a binary that should exist.

**Comment by damienmg** [comment]
This test has been deleted by #18.

**Comment by katre** [comment]
Thanks!

**Comment by ittaiz** [comment]
@katre you'll close this after you see its green?

**Comment by katre** [comment]
Yup. Still waiting for a few fixes to cut the next RC, but I'll take care
of closing this. Thanks for helping!

**Comment by katre** [comment]
Fixed in 0.11.0rc4.
